### ISSUE TITLE
Configurable Entry Key Generator

### ISSUE THREAD

**Issue by inukshuk** [inukshuk/bibtex-ruby#20]
A Configurable Entry Key Generator

**Comment by mapreal19** [comment]
@inukshuk hijacking this issue just to mention about maybe using a bot for stale issues: https://github.com/apps/stale ? 

I wish I had more time for helping, but at least we could focus on the most important issues

**Comment by inukshuk** [comment]
Good idea, but I think while we have only a handful issues (only 13 open issues at the moment), we can just use tags or close issues manually.
